### PR TITLE
Accordion

### PR DESCRIPTION
**Pull request by kingasieminiak**
Created structure for accordion and card components.
Add basic styles.

### CODE REVIEW

**Comment by tborowy** [comment]
Create PR to develop, not master

File: src/assets/icons/icon_4.svg [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                <g id="Group-5">
+                    <g id="Oval-6">
+                        <use fill="black" fill-opacity="1" filter="url(#filter-11)" xlink:href="#path-10"></use>
+                        <use stroke="url(#linearGradient-9)" stroke-width="9" xlink:href="#path-10"></use>
+                    </g>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (tborowy) [inline src/assets/icons/icon_4.svg]: add new line

File: src/assets/icons/icon_1.svg [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                    </g>
+                    <g id="Rectangle-9">
+                        <use fill="#FFFFFF" fill-rule="evenodd" xlink:href="#path-7"></use>
+                        <rect stroke="#567BE2" stroke-width="2" x="22" y="90" width="7" height="7" rx="3.5"></rect>
+                    </g>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (tborowy) [inline src/assets/icons/icon_1.svg]: add new line

File: src/assets/icons/icon_2.svg [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                            <use fill="black" fill-opacity="1" filter="url(#filter-7)" xlink:href="#path-6"></use>
+                            <use stroke="#567BE2" stroke-width="2" xlink:href="#path-6"></use>
+                        </g>
+                    </g>
+                    <path d="M78,156 C121.07821,156 156,121.07821 156,78 C156,34.9217895 121.07821,0 78,0" id="Oval-6" stroke="url(#linearGradient-8)" stroke-width="9"></path>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (tborowy) [inline src/assets/icons/icon_2.svg]: add new line

File: src/assets/icons/icon_3.svg [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+                                <path d="M0.479591837,36 L32.6122449,36" id="Line-5"></path>
+                            </g>
+                        </g>
+                    </g>
+                    <path d="M5,88 L34.8659841,64.0109579 L34.8659841,64.0109579 C42.6118115,57.789332 53.657743,57.8471177 61.3380526,64.1494438 L90,87.6689306 L5,88 Z" id="Path-4" stroke="#567BE2" stroke-width="2" fill="#DEE3EC"></path>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (tborowy) [inline src/assets/icons/icon_3.svg]: add new line

File: src/components/accordionItem.component.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+class AccordionItem extends Component {
+    constructor() {
+        super();
+        this.state = {
+            isOpen: false
+        };
+    }
+
+    toggleAccordionItem = () => {
```
Comment (tborowy) [inline src/components/accordionItem.component.js]: use `toggleAccordionItem() {...}` not `toggleAccordionItem = () => {...}`

File: src/components/main.js [hunk 6]
```diff
@@ -38,14 +40,31 @@ class Main extends Component {
...
+                    </section>
                 </div>
+
+                <div className='background'>
+                    <section className='segment'>
+                        <h2 className='segment__title segment__title--centered'>
+                            Wszystko, co powinieneś wiedzieć o aplikacji
+                        </h2>
+
+                        <Accordion />
```
Comment (tborowy) [inline src/components/main.js]: I think... this module should receive one property with the array of objects... one object defines one accordionItem element.
In module should use a loop on this array and generate <AccordionItem> with two params... title and description from this array.
One AccordionItem should use this param to generate view.

**Comment by tborowy** [comment]
delete merge commit (8c4b5f6) this commit should be in another branch and in new PR to develop.

**Comment by tborowy** [comment]
What is a Card component? Probably this component should be in another PR from another branch

**Comment by tborowy** [comment]
good practice to the rest of the elements should be as one commit because all the changes concern one functionality

File: src/components/main.js (hunk 6 above)
Comment (kingasieminiak) [reply src/components/main.js]: I agree with you. What solution can you recommend to keep text data in one place?

File: src/components/accordionItem.component.js (hunk 5 above)
Comment (kingasieminiak) [reply src/components/accordionItem.component.js]: Are you sure it is the best option? How can I have access to 'this.state' when I use toggleAccordionItem() {...}?

**Pull request closed without merging**